### ISSUE TITLE
sorting imports issue

### ISSUE THREAD

**Issue by willt** [FractalBoy/perl-language-server#68]
Sorting imports doesn't seem to work correctly for me.

If there is a blank line or comment it stops processing anything after the comment or blank line. It also doesn't seem to be sorting things alphabetically properly.

I see you are using ExtUtils::Installed to see what modules are installed. Sadly this doesn't really work for centos and ubuntu based systems and the likes because they don't include the .packlist in their rpm/dep packages. I only have packlists on my system for stuff I've installed out of cpan.
    
I'd either make using ExtUtils::Installed optional or remove it. I don't think it's gonna do a whole lot for most people. I don't know of any other better solutions. Maybe examine what's in @INC ? I think that's what https://github.com/briandfoy/app-module-lister does.

Personally I don't care for the spacing between the sections. I'd like to see an option to disable/enable spacing between sections. 

What are your thoughts?

**Comment by FractalBoy** [comment]
Yeah, I never made this feature particularly robust. It was originally just meant to sort things how I like them. It stops after the first non "use" line. Maybe that's not necessary, I just never tried to make it too complicated.

Perhaps if that option is available, then pragmas can stay in their own sections and all other modules can just be combined and sorted.

**Comment by willt** [comment]
A side effect of having them stop after the first non use line is that once
you sort them, you won't be able to sort them again.
I think just sorting them would probably be good enough with pragmas on
top. Ideally I think it would go:
1 pragmas
2 installed modules (any module outside of project root)
3 modules found in project root

The other related thing I forgot to ask is, shouldn't this feature be
provided via a code action ? I was a little confused in
reading whether it should or shouldn't be.
